src/command_modules/azure-cli-network/azure/cli/command_modules/network/commands.py

```diff
@@ -7,7 +7,7 @@
 
 from azure.cli.core.commands.arm import cli_generic_update_command, cli_generic_wait_command
 from azure.cli.core.commands import DeploymentOutputLongRunningOperation, cli_command
-from ._client_factory import * #pylint: disable=wildcard-import
+from ._client_factory import * # pylint: disable=wildcard-import, unused-wildcard-import
```

Comment:
One of the imported symbols is not used in this file, but it is used elsewhere. Rather than import the 30 or so other symbols individually, this seemed the more practical approach.